### PR TITLE
Fixes, and changes for Dapp styling

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
This PR it's to improve the styling in general of the Dapp

**New stuff** ✨

* Added IBMPlexMono (techie font) SemiBold
* Added a different version of the caret-right icon for the right caret in the UAC
* Added correct colors and weight to InfoPopover heading, error, and etherscan link
* Added separator between "New Action" button and domain dropdown, and the event list in the Colony Home

**Changes** 🏗
* Improved alignment of copyable address
* Reduced width of Dialog core component
* Increased margin between Dialog outer actions and Dialog main content
* Updated color of Select core component to correct one
* Fixed extraTiny className not being applied to Icon
* Added correct font weight and color to MaskedAddress
* Updated Modal overlay color
* Reduced width of SinglePickerUser
* Updated Navbar text color
* Updated active item color of domain dropdown
* Updated font weight of Permission Management checkbox
* Increased width of TokenAmount second column in Transfer Funds Dialog
* Decreased padding of User Notification button

**Deletions** ⚰️
* Removed incorrect font (techie) from TokenSelector

Contributes to DEV-134

### CODE REVIEW

**Comment by rdig** [comment]
Even though this is still a draft, please tag the linear issue this is fixing.

Thanks! :+1:

File: src/img/icons.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "caret-left",
     "caret-right",
     "caret-up-small",
+    "caret-right-small",
```
Comment (alicjakujawa) [inline src/img/icons.json:13]: this need to be sorted alphabetically, just move it above :)

File: src/img/icons/caret-right-small.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<svg width="6" height="10" viewBox="0 0 6 10" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0)">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M0.469828 9.88039C0.569828 9.98039 0.679828 10.0304 0.729828 9.98039L5.51983 5.20038C5.53674 5.18347 5.53935 5.15225 5.52917 5.11429C5.53509 5.07969 5.52923 5.04979 5.50985 5.03041L5.42789 4.94862C5.42525 4.94586 5.42256 4.94312 5.41983 4.94038L5.41226 4.93303L0.719846 0.250411C0.679846 0.21041 0.559846 0.250411 0.459846 0.350411L0.119846 0.700409C0.0198461 0.80041 -0.0301541 0.91041 0.0198456 0.96041L4.17917 5.11105L0.0198278 9.27039C-0.0201722 9.31039 0.0198283 9.43039 0.119828 9.53039L0.469828 9.88039Z" fill="#3C444D" fill-opacity="0.7"/>
+</g>
+<defs>
+<clipPath id="clip0">
+<rect width="5.53" height="9.77" fill="white" transform="translate(5.52979 10) rotate(180)"/>
+</clipPath>
+</defs>
+</svg>
```
Comment (alicjakujawa) [inline src/img/icons/caret-right-small.svg:10]: This svg needs to be updated @rdig  maybe you could do call with full dapp team about better svg optimisation ?

File: src/modules/core/components/InfoPopover/TokenInfo.tsx [hunk 3]
```diff
@@ -48,7 +47,7 @@ const TokenInfo = ({ token, isTokenNative }: Props) => {
           </p>
         )}
         <div title={address} className={styles.address}>
-          <CopyableAddress full>{address}</CopyableAddress>
+          {address}
```
Comment (alicjakujawa) [inline src/modules/core/components/InfoPopover/TokenInfo.tsx:50]: Are we sure that we dont make this copyable?

File: src/styles/variables.css [hunk 4]
```diff
@@ -114,11 +114,12 @@
   --spacing-medium: 0.5px;
 
   /**
-   * @NOTE Weight medium is olny available on the techie font family
+   * @NOTE Weight medium and semi bold is olny available on the techie font family
```
Comment (alicjakujawa) [inline src/styles/variables.css:117]: typo

**Review by alicjakujawa — CHANGES_REQUESTED** [review]
Small comments, but overall nice one 🔥

File: src/img/icons.json (hunk 1 above)
Comment (chinins) [inline src/img/icons.json:13]: I would move it 2 lines up - right before `caret-right`, so it's grouped together and preserves alphabetical order

File: src/img/icons/caret-right-small.svg [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<svg width="6" height="10" viewBox="0 0 6 10" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (chinins) [inline src/img/icons/caret-right-small.svg:1]: I think, usually `viewBox` should be set to `0 0 30 30` to have consistency throughout the app (you would need to change the whole svg though)

File: src/styles/variables.css (hunk 4 above)
Comment (chinins) [inline src/styles/variables.css:117]: typo in `olny`

**Review by chinins — CHANGES_REQUESTED** [review]
Just a couple of small changes.

File: src/img/icons/caret-right-small.svg (hunk 2 above)
Comment (chinins) [reply src/img/icons/caret-right-small.svg:10]: I think it's a great idea. I would appreciate that.

File: src/modules/core/components/InfoPopover/TokenInfo.tsx (hunk 3 above)
Comment (ArmandoGraterol) [reply src/modules/core/components/InfoPopover/TokenInfo.tsx:50]: Yeah... Now that I think more about it, I don't see why not. I removed it initially because in the design it doesn't really look copyable

**Review by rdig — CHANGES_REQUESTED** [review]
@ArmandoGraterol There's a bunch of "merge" commits in your branch.

Screenshot from 2021-01-06 19-47-41

I think you either didn't rebase properly or have had a merge conflict that wasn't resolved properly.

Please re-write this branch history to reflect just the changes you've done here.

Thanks.

**Comment by rdig** [comment]
Also, please try to be more specific with your commit messages:

Screenshot from 2021-01-06 19-49-46

The ones like the above don't provide much information to the reader, and months from now when we're looking through the commit history for a specific change, it will make our life very hard trying to decipher commits like this one.

File: src/modules/users/components/AvatarDropdown/AvatarDropdown.tsx [hunk 6]
```diff
@@ -66,13 +67,19 @@ const AvatarDropdown = ({ onlyLogout = false }: Props) => {
...
+            [styles.activeDropdown]: isOpen,
+          })}
+          onClick={toggle}
+          type="button"
+          data-test="avatarDropdown"
+          {...props}
+        >
+          <UserAvatar address={walletAddress} notSet={ethereal} size="s" />
+        </button>
+      )}
```
Comment (rdig) [inline src/modules/users/components/AvatarDropdown/AvatarDropdown.tsx:82]: You introduce some react standardization errors in here:

Screenshot from 2021-02-05 14-34-56

Please investigate and fix it.

File: src/modules/pages/components/RouteLayouts/UserNavigation.css [hunk 7]
```diff
@@ -12,7 +12,7 @@
 .notificationsIcon {
   flex-grow: 0;
   margin-right: 15px;
-  padding: 7px 9px 9px;
+  padding: 7px 6px;
```
Comment (rdig) [inline src/modules/pages/components/RouteLayouts/UserNavigation.css:15]: This (or at least I assume it was this), broke the user notification dot:

Screenshot from 2021-02-05 14-35-21

File: src/modules/core/components/Fields/SelectOption/SelectOption.css [hunk 8]
```diff
@@ -4,23 +4,27 @@
...
+  color: color-mod(var(--dark) alpha(50%));
+  pointer-events: none;
+}
 
-  &[aria-disabled="true"] {
-    color: color-mod(var(--grey-1) alpha(50%));
-    pointer-events: none;
-  }
+.main[data-checked="true"] {
+  color: var(--primary);
```
Comment (rdig) [inline src/modules/core/components/Fields/SelectOption/SelectOption.css:27]: Why was this introduced? 

This colors all checkboxes default selected options as "green" (our "primary" color)

Screenshot from 2021-02-05 14-35-33

Screenshot from 2021-02-05 14-35-37

Screenshot from 2021-02-05 14-36-21


If this was done for a single specific select box then please refactor this, otherwise, I don't think this is a good change.

File: src/modules/dashboard/components/CreateColonyWizard/TokenSelector.css [hunk 9]
```diff
@@ -1,4 +1,3 @@
 .inputWrapper > div > div > input {
-  font-family: var(--family-techie);
   font-weight: var(--weight-medium);
 }
```
Comment (rdig) [inline src/modules/dashboard/components/CreateColonyWizard/TokenSelector.css:3]: Why was this removed?

@alicjakujawa specifically added this at some point, and I also believe it was cleared with Karol.

Screenshot from 2021-02-05 14-36-51

File: src/modules/core/components/SingleUserPicker/SingleUserPicker.css [hunk 10]
```diff
@@ -57,7 +57,7 @@
 .baseInput {
   flex: 1 1 0;
   margin-left: 10px;
-  max-width: 340px;
+  max-width: 305px;
```
Comment (rdig) [inline src/modules/core/components/SingleUserPicker/SingleUserPicker.css:60]: Why was this reduced ?

If it's just for single instance do it in the spefic component, otherwise, I don't this change is appropriate _(as user addresses tend to be quite long)_

File: src/modules/core/components/MembersList/MembersListItem.tsx [hunk 11]
```diff
@@ -193,9 +193,9 @@ const MembersListItem = <U extends AnyUser = AnyUser>(props: Props<U>) => {
               <UserMention hasLink={false} username={username} />
             </span>
           )}
-          <span className={styles.address}>
-            <MaskedAddress address={walletAddress} />
-          </span>
+          <div className={styles.address}>
+            <CopyableAddress>{colonyAddress}</CopyableAddress>
+          </div>
```
Comment (rdig) [inline src/modules/core/components/MembersList/MembersListItem.tsx:198]: Are we totally sure this change is correct?

Screenshot from 2021-02-05 15-08-05

I feel like the previous version looked better imho

File: src/modules/pages/components/RouteLayouts/Default/Default.css [hunk 12]
```diff
@@ -9,7 +9,7 @@
 
 .coloniesList {
   flex: 0 0 auto;
-  width: 50px;
+  width: 60px;
```
Comment (rdig) [inline src/modules/pages/components/RouteLayouts/Default/Default.css:12]: I feel like this is a bad change. 

Note that increasing this has a cascading effect over the whole layout of the app as a lot of pages do width `calc` assuming this 50px. So if you change it here make sure you redo all those other places that this influences.

Right now, it broke the full page loading layout:

Screenshot from 2021-02-05 15-12-56

Notice the scroll bar at the bottom!

File: src/styles/variables.css [hunk 13]
```diff
@@ -114,11 +114,12 @@
   --spacing-medium: 0.5px;
 
   /**
-   * @NOTE Weight medium is olny available on the techie font family
+   * @NOTE Weight medium and semi bold is only available on the techie font family
    */
   --family-monospace: "Source Code Pro", "Monaco", "Inconsolata", monospace;
   --family-techie: "IBM Plex Mono", sans-serif;
   --weight-medium: 500;
+  --weight-semi-bold: 600;
```
Comment (rdig) [inline src/styles/variables.css:122]: I really don't feel good about adding a new font file _(which we have to distribute)_ for a font-weight that is so minor in size from the previous one....

**Review by rdig — CHANGES_REQUESTED** [review]
This still needs a lot of work as, from my point of view, it broke more things than it fixed.

So please take another look at it, fix the things that I've pointed out. 

The thing to keep in mind is, that if _product_ requests a change you should **always** apply your own judgement first, before making that change. 

I'm talking here about adding an extra font file, just for that minor font weight. This is 100K that we're adding to each download of app bundle, each time, for just a few places where we use it. I feel like the tradeoff in "polish" is not worth it in this case.

For future reference please bring issues like this to the attention of the team so that we can decide, as a group, on the best course of action.

File: src/modules/core/components/Fields/SelectOption/SelectOption.css (hunk 8 above)
Comment (ArmandoGraterol) [reply src/modules/core/components/Fields/SelectOption/SelectOption.css:27]: This was introduced due to this: `Selected domain colour (dropdown) is wrong (consistent behaviour)` being in vol.1 of the feedback provided by Karol and Jack. If I remember correctly, I checked the style guide in Figma, and I saw that the selected option colour is the primary colour.

Do you think it's better in the old colour?

File: src/modules/dashboard/components/CreateColonyWizard/TokenSelector.css (hunk 9 above)
Comment (ArmandoGraterol) [reply src/modules/dashboard/components/CreateColonyWizard/TokenSelector.css:3]: I removed it due to this change referencing this input: `Use of wrong typography for input content, which is, misaligned`

It's in vol.2 of the changes. I think this doc in particular was written by Jack right? That would explain why I was asked to change the typo even though you guys talked with Karol about it

File: src/modules/core/components/SingleUserPicker/SingleUserPicker.css (hunk 10 above)
Comment (ArmandoGraterol) [reply src/modules/core/components/SingleUserPicker/SingleUserPicker.css:60]: I think, if I remember correctly, it was reduced because of this: `Use of inconsistent inputs (modals) - long width, wrong height`

Should I change it back?

File: src/modules/core/components/MembersList/MembersListItem.tsx (hunk 11 above)
Comment (ArmandoGraterol) [reply src/modules/core/components/MembersList/MembersListItem.tsx:198]: In vol.2: `wallet address should be possible to copy-paste`

I agree with you in that the previous version looked better tho

File: src/modules/pages/components/RouteLayouts/Default/Default.css (hunk 12 above)
Comment (ArmandoGraterol) [reply src/modules/pages/components/RouteLayouts/Default/Default.css:12]: I will see if adjusting all the other places isn't too much problem then

File: src/styles/variables.css (hunk 13 above)
Comment (ArmandoGraterol) [reply src/styles/variables.css:122]: Let's remove it for now, right now it's just 1 place where it's used

File: src/modules/pages/components/RouteLayouts/Default/Default.css (hunk 12 above)
Comment (ArmandoGraterol) [reply src/modules/pages/components/RouteLayouts/Default/Default.css:12]: Tested it at 50px, it's not this change that's breaking the page

Comment (ArmandoGraterol) [reply src/modules/pages/components/RouteLayouts/Default/Default.css:12]: Hell, lemme tell you something. This is even happening in `master` lol

Tried to look at master to see where's the difference, but there's no difference

File: src/modules/core/components/SingleUserPicker/SingleUserPicker.css (hunk 10 above)
Comment (rdig) [reply src/modules/core/components/SingleUserPicker/SingleUserPicker.css:60]: Nah, leave it.

File: src/modules/pages/components/RouteLayouts/Default/Default.css (hunk 12 above)
Comment (rdig) [reply src/modules/pages/components/RouteLayouts/Default/Default.css:12]: I'll test it again

File: src/modules/dashboard/components/CreateColonyWizard/TokenSelector.css (hunk 9 above)
Comment (rdig) [reply src/modules/dashboard/components/CreateColonyWizard/TokenSelector.css:3]: But we discussed this with Karol at some point, and agreed that techie was better here. 

Please change it back

File: src/modules/core/components/Fields/SelectOption/SelectOption.css (hunk 8 above)
Comment (rdig) [reply src/modules/core/components/Fields/SelectOption/SelectOption.css:27]: Nah leave it, don't bother.

File: src/modules/pages/components/RouteLayouts/Default/Default.css (hunk 12 above)
Comment (rdig) [reply src/modules/pages/components/RouteLayouts/Default/Default.css:12]: Yeah, you can leave it as it is for now, I'll test it if something actually breaks when I rebase my #2359 then I'll fix it there

**Review by rdig — APPROVED** [review]
Good to go

File: src/modules/dashboard/components/CreateColonyWizard/TokenSelector.css (hunk 9 above)
Comment (ArmandoGraterol) [reply src/modules/dashboard/components/CreateColonyWizard/TokenSelector.css:3]: Sure :+1:

**Review by alicjakujawa — APPROVED** [review]
🔥
